kickstarts/partials/packages/excludes.ks.erb

```diff
@@ -9,15 +9,23 @@
 -ipw2200-firmware
 -ivtv-firmware
 -iwl100-firmware
+-iwl105-firmware
+-iwl135-firmware
 -iwl1000-firmware
+-iwl2000-firmware
+-iwl2030-firmware
+-iwl3160-firmware
 -iwl3945-firmware
 -iwl4965-firmware
 -iwl5000-firmware
 -iwl5150-firmware
 -iwl6000-firmware
 -iwl6000g2a-firmware
+-iwl6000g2b-firmware
 -iwl6050-firmware
-#kernel-firmware
+-iwl7260-firmware
+-iwl7265-firmware
+-kernel-firmware
```

Comment:
I looked at all the hardware that is supposed by these firmwares (including kernel-firmware), and there isn't a hypervisor that implements the hardware as a virtual device. I checked both ESX and QEmu.

Can I guarantee that there isn't an exotic situation where these firmwares are needed in a virtual appliance? No... But IMHO it's very unlikely, and therefore I think these should be removed. In the unlikely event we'd presumable get a good error message from the kernel driver and we can put it back.